src/reynir_correct/pattern.py

```diff
@@ -1646,15 +1636,16 @@ def create_patterns(cls) -> None:
                 )
             )
 
-            # Catch "Ég hafði ekki lagt mikið að mörkum."
+            # Catch "Ég lagði mikið að mörkum."
             p.append(
                 (
-                    "mark",  # Trigger lemma for this pattern
-                    "VP > { VP >> { 'leggja' } PP > { P > 'að' NP > { 'mark' } } }",
+                    "mörk",  # Trigger lemma for this pattern
```

Comment:
"mörk" and "mark"? It is possible to have multiple trigger lemmas.

Reply:
Done!